tests/drawer/test_tape_mpl.py

```diff
@@ -539,6 +535,37 @@ def test_general_operations(self, op):
 
         plt.close()
 
+    def test_snapshot(self):
+        """Test that `qml.Snapshot` works properly with `tape_mpl`."""
+
+        # Test that empty figure is created when the only gate is `qml.Snapshot`
+        with qml.queuing.AnnotatedQueue() as q_tape:
+            qml.Snapshot()
+
+        tape = QuantumScript.from_queue(q_tape)
```

Comment:
we often use queuing unnecessarily in tests, but it's too hard to fix existing ones. but we might as well make new test code cleaner!
```suggestion
        tape = QuantumScript([qml.Snapshot()])
```